packages/swipe-carousel/package.json

```diff
@@ -1,27 +1,16 @@
 {
   "name": "@pokeberry/swipe-carousel",
   "private": false,
-  "version": "0.0.1",
-  "files": [
-    "dist",
-    "README.md",
-    "LICENSE"
-  ],
-  "main": "./dist/swipe-carousel.umd.js",
-  "module": "./dist/swipe-carousel.es.js",
-  "exports": {
-    ".": {
-      "require": "./dist/swipe-carousel.umd.js",
-      "import": "./dist/swipe-carousel.es.js"
-    }
-  },
+  "version": "0.1.0",
+  "main": "dist/index.ts",
```

Comment:
wouldn't these still be `.js` files?

(these `export` formats came from `vite` documentation. i just forgot to reimplement splitting it into two different formats -- `ejs` and `umd`)

Reply:
yeah, sure! it should be js, and I'll take a look to rebasing it now